chunky/src/java/se/llbit/chunky/world/World.java

```diff
@@ -131,12 +96,61 @@ public enum LoggedWarnings {
     SILENT
   }
 
+  public void loadDimension(int dimensionId) {
+    currentDimension = loadDimension(this, this.worldDirectory, dimensionId, -1, Collections.emptySet());
+    currentDimension.reloadPlayerData();
+  }
+
+  /**
+   * Add a chunk deletion listener.
+   *
+   * @deprecated Remove in 2.7.
+   * Replaced by {@link World#currentDimension()} -> {@link Dimension#addChunkDeletionListener}
+   */
+  @Deprecated
```

Comment:
It didn't have `@PluginApi`, so you could as well just delete these methods.

If you don't, it's probably fine to remove them in 2.6. Are there any plugins that actually use these APIs?

Reply:
Beyond my own, no I don't think so. I'll remove it :+1: